### PR TITLE
Prepare for hacktoberfest

### PR DESCRIPTION
**Pull request by Sheeri**

### CODE REVIEW

**Comment by Sheeri** [comment]
Prepare for hacktoberfest and updating typo in "GROOP".

**Review by Azzoo — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Fork the repo
+Clone the repo
+Track changes
+Create a new branch on your fork
+Change and TEST your code
+Commit and push
+Create a pull request
+Wait for a review
+Fix anything the review brought up
+Celebrate when the code gets merged!
```
Comment (am-MongoDB) [inline CONTRIBUTING.md:15]: If this is meant to be a bulleted list, then should add `-` to the start of each line

Comment (Sheeri) [reply CONTRIBUTING.md]: @am-MongoDB updated!

**Review by am-MongoDB — APPROVED** [review]

**Comment by Sheeri** [comment]
@jsdmaria could you give a quick review? I had some conflicts, but I believe I resolved them.

**Pull request merged**